### ISSUE TITLE
New version terribly slow and unresponsive after slicing (Mac)

### ISSUE THREAD

**Issue by aayevtushenko** [bambulab/BambuStudio#5200]
### Bambu Studio Version

1.10.0.89

### Where is the application from?

Bambu Lab Official website

### OS version

macOS Sonoma 14.7

### Additional system information

Processor: 2.6GHz 6-Core Intel Core i7
Memory: 16GB
macOS: Sonoma 14.7

### Printer

Bambu A1

### How to reproduce

Slice this model https://makerworld.com/en/models/746690#profileId-679824
Try moving camera, clicking menus, etc

Converting back to the previous stable version resolves the issue.

### Actual results

Any action takes 7+ seconds to execute

### Expected results

like with the previous version actions should work without painful lag

### Project file & Debug log uploads

log file is not available at application support/bambustudio.
[Retractable_Cable_Organizer_V2.3mf.zip](https://github.com/user-attachments/files/17720618/Retractable_Cable_Organizer_V2.3mf.zip)

### Checklist of files to include

- [X] Log file
- [X] Project file

**Comment by zanuda2raza** [comment]
Same MBP 16, i9, macOS 15.1

**Comment by FEMit160** [comment]
Seems to work the same (good) as prior versions for me. Windows 10, i7-9th gen, 16GB ram, GF GTX 1650 4gb mobile.

**Comment by mikegabriel** [comment]
Based on comments I have read I think it's an Intel / Mac thing.  Over on the reddit subreddit someone replied to my comment that it's fine on their M1 Pro.

For the record, I am on a 2019 MBP with an Intel I9 and 32gb ram, 8gb ram radon 5500m.  

Rolling back to previous version resolved the issue.

**Comment by Cyberdoc** [comment]
Same here: 2019 MBP, i9, 64GB RAM, 8GB 5500m - macOS 15.0.1

Also back to previous version, which "removed" the problem.

**Comment by okovpashko** [comment]
I just updated to v1.10 and noticed the same issue.
2023 MacBook Pro M3 Max, 36 GB RAM, macOS Sonoma 14.16.1.

Also, it seems like the organic tree support generation is broken as it generates the default "normal" support.

Had to roll back to v1.9.7.52 where everything works fine.

**Comment by fah** [comment]
iMac Intel/2019: Same problem. I only see the ball spinning every time I move the cursor. Not usable!
V 1.10.0.89

**Comment by killsecurly** [comment]
This all seems to be a Intel Mac issue. My MBA M1 works fine with v1.10.

**Comment by CodeByZach** [comment]
Same exact issue here. Slicing makes it unusable. Had to revert.

**Comment by lanewei120** [comment]
the same as https://github.com/bambulab/BambuStudio/issues/5194

**Comment by Haidiye00** [comment]
Hello, we suspect it may be a problem with the graphics card or graphics card driver. Could you please provide your graphics card model and driver? Thank you

**Comment by lanewei120** [comment]
Hello, @mikegabriel @Cyberdoc @CodeByZach @okovpashko @aayevtushenko @zanuda2raza @FEMit160 @killsecurly 
thanks for reporting this issue

we have tried our best to reproduce this issue here,
but still can not reproduce it

could someone also help to record a short video for the detailed steps
and help to upload the log here(https://wiki.bambulab.com/en/x1/troubleshooting/bambu-studio-log)
just like [northdakota](https://github.com/northdakota) did in issue 5194

we need to collect more details to dig into this issue

**Comment by Haidiye00** [comment]
![img_v3_02gj_0d8f6d82-9408-49fc-904c-988c7e744c7g](https://github.com/user-attachments/assets/7b1a16a9-962f-414d-93a0-bd2d25597fcc)
![img_v3_02gj_d9234102-114f-424d-805f-4108d4e3c27g](https://github.com/user-attachments/assets/0e6d3aec-35f3-4da5-8625-b138bad03daa)

Hello, can you take a screenshot according to this? Thank you

**Comment by tangketan** [comment]
@okovpashko As to the support issue, did you use variable layer height? Organic support doesn't work with variable layer height, so we defaulted to hybrid tree support, which may generate "normal-like" structures.

**Comment by Haidiye00** [comment]
![img_v3_02gj_8b065a80-fcbe-4f51-b833-dcee96824e0g](https://github.com/user-attachments/assets/33fab8e1-1f27-4a99-a330-0179dcd1d840)
![img_v3_02gj_8a9692e8-7464-4671-a5bd-5751a1eb01ag](https://github.com/user-attachments/assets/76936360-a4e4-4ae3-9719-1219bd6b6d60)
Hello, can you help test this method? Follow the screenshot to close the two boxed options, then restart the software and help test if there are still any issues? thank you

**Comment by Haidiye00** [comment]
https://github.com/bambulab/BambuStudio/releases/tag/v01.09.07.52
Could you please test if this version is working properly? Thank you

**Comment by TransparentHouseSF** [comment]
> https://github.com/bambulab/BambuStudio/releases/tag/v01.09.07.52 Could you please test if this version is working properly? Thank you

this old version is working fine. 1.10.0.89 is not.

**Comment by okovpashko** [comment]
@tangketan I'm sorry, I mixed up the organic tree support with the hybrid. The organic tree works as expected. The hybrid tree in v1.10 is generated as "normal" whereas v1.9.7 generates it as the "tree". When I open the same project with these two versions of Bambu Studio I see different support on the preview tab.

I didn't find any related changes in the changelog so I'm wondering if it's an expected behavior.
Should I open a separate issue for that?

**Comment by okovpashko** [comment]
As for the main issue of the current topic, I'm confused now.
Yesterday I definitely had a slow and irresponsive rendering on the preview tab in v1.10 and replaced it with 1.9.7 where everything worked well. Today I installed v1.10 in parallel to v1.9.7 and there are no issues with the preview at all 🤔 

The v1.10 was installed on a fresh OS that never had the previous Bambu Studio versions.

**Comment by mkim1** [comment]
I'm also having this issue with the following device. Downgrading to v01.09.07.52 resolved the issue for me.
<img width="698" alt="image" src="https://github.com/user-attachments/assets/dc436ad1-2539-4999-8fd1-63a13a70f6ef">

**Comment by mkim1** [comment]
I'm also having this issue with the following device. Downgrading to v01.09.07.52 resolved the issue for me.
<img width="263" alt="image" src="https://github.com/user-attachments/assets/b43d9e46-ea9a-4390-9e6c-cbaaa9132ab3">

**Comment by mikegabriel** [comment]
I reinstalled 1.10.0.89 this morning and did a bit more testing and for me it does seem to take a significant performance drop the second I add an additional plate.  

If I start with an empty file, right click the build plate and add a primitive -> 3D benchy and then slice, everything performs normally.

Now If I click Add Plate and then reclick my first plate so it's active, drag the benchy so that I can click Slice plate again, it's completely unusable in the preview tab and beachballs.

![Screenshot 2024-11-13 at 9 28 54 AM](https://github.com/user-attachments/assets/a58d00d1-5cfa-49b6-a1b6-364cad95fd63)

**Comment by Malrikgus** [comment]
Same issue here. Monterey 12.7.

**Comment by QZBT** [comment]
The same situation occurs on Mac with Intel CPU.
When slicing is completed, staying on the preview interface will become unresponsive.
If you can click Return to the Preparation Page, it will return to normal.
Temporary solution: Click Return to the Preparation Page as soon as possible before slicing is completed.

**Comment by Batou99** [comment]
Same here, iMac with intel i5, Radeon Pro 580X 8 GB. Completely unusable.

**Comment by MartinChan3** [comment]
It seems that Studio has released a temporary package to provide an interim solution for those experiencing issues on Mac (Intel chip), STEP import crashes, and missing temperature displays. Hopefully, they will release the actual hotfix soon!
Windows：https://github.com/bambulab/BambuStudio/releases/download/v01.10.00.89/Bambu_Studio_win_public-v01.10.00.94-20241114223322.exe
Mac：https://github.com/bambulab/BambuStudio/releases/download/v01.10.00.89/Bambu_Studio_mac-v01.10.00.94-20241114223322.dmg

**Comment by aayevtushenko** [comment]
the hotfix provided in this other issue seems to have resolved the issue.
https://github.com/bambulab/BambuStudio/issues/5194#issuecomment-2476700409

**Comment by aayevtushenko** [comment]
Can this issue be merged into [a duplicate](https://github.com/bambulab/BambuStudio/issues/5194#issue-2652498309)?

**Comment by HO44-PROJECT** [comment]
They just released 1.10.1.50 that closes my similar case.
https://github.com/bambulab/BambuStudio/issues/5194
